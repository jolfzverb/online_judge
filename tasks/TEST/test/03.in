-13131
12331
5552234
42
2398463854782634
