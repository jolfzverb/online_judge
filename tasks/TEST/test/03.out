-13131
12331
5552234
